Utilities/drone_exceptions.py

```diff
@@ -0,0 +1,71 @@
+class NetworkException(Exception):
+    """
+    Any network or connection errors
+    """
+    pass
+
+
+class AltitudeException(Exception):
+    """
+    Too low or too high altitude
+    """
+    pass
+
+
+class ThrustException(Exception):
+    """
+    Too low or too high thrust
+    """
+    pass
+
+
+class VelocityException(Exception):
+    """
+    Too low or too high velocity
+    """
+    pass
+
+
+class BadArgumentException(Exception):
+    """
+    Caused by wrong call to function or something
```

Comment:
Not very professional to add `or something` here. Also, this should be a bit more explicit about its purpose rather than being the same as `EmergencyLandException`. Not only that, the Python standard exception for an invalid argument value is `ValueError`, so consider using that in place of your own class.

Reply:
I overlooked the docstring of BadArgumentException. I have removed it. The EmergencyLandException docstring must have been copied from BadArgumentException and never changed. I have replaced its docstring with an appropriate one.